Makefile

```diff
@@ -598,3 +602,20 @@ certs:
 .PHONY: certs-dryrun
 certs-dryrun:
 	cd pkg/config/tlscfg/testdata && ./gen-certs.sh -d
+
```

Comment:
are these targets necessary?

Reply:
I think having repro-check is useful to give everything a whirl without thinking too hard about how to do that. That said, the collect and test sums targets could be inlined there.